src/complexity.cc

```diff
@@ -208,11 +212,13 @@ std::vector<BenchmarkReporter::Run> ComputeBigO(
   // Only add label to mean/stddev if it is same for all runs
   Run rms;
   rms.run_name = run_name;
-  big_o.report_label = reports[0].report_label;
   rms.run_type = BenchmarkReporter::Run::RT_Aggregate;
   rms.aggregate_name = "RMS";
   rms.report_label = big_o.report_label;
   rms.iterations = 0;
+  rms.repetition_index = -1;
```

Comment:
given you use -1 as a sentinel, you might want to document this at the declaration of `repetition_index` in `Run`.

https://google.github.io/styleguide/cppguide.html#Variable_Comments